Example/Messaging/App/watchOS/SampleWatchApp/SampleWatchAppWatchKitExtension/ExtensionDelegate.swift

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import WatchKit
+
+import FirebaseCore
+import FirebaseMessaging
```

Comment:
Is there a reason we can't use `import Firebase` here? If not, we should use it :)

Reply:
I'm getting 'No such module 'Firebase'' error.
Maybe I need to update my podfile to have pod 'firebase'?